tests/bootstrap.php

```diff
@@ -2,53 +2,4 @@
 $wgRunningUnitTests = true; // don't include DevBoxSettings when running unit tests
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)